src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -112,26 +114,21 @@ contract BancorVirtualSupplyBondingCurveFundingManager is
         __Module_init(orchestrator_, metadata);
 
         address _acceptedToken;
-        IssuanceToken memory issuanceToken;
+        address _issuanceToken;
         BondingCurveProperties memory bondingCurveProperties;
 
-        (issuanceToken, bondingCurveProperties, _acceptedToken) = abi.decode(
-            configData, (IssuanceToken, BondingCurveProperties, address)
-        );
+        (_issuanceToken, bondingCurveProperties, _acceptedToken) =
+            abi.decode(configData, (address, BondingCurveProperties, address));
 
-        __ERC20_init(
-            string(abi.encodePacked(issuanceToken.name)),
-            string(abi.encodePacked(issuanceToken.symbol))
-        );
+        // Set issuance token. This also caches the decimals
+        _setIssuanceToken(_issuanceToken);
```

Comment:
This change seems to introduce functionality such that the FM_BC **only** works with an external ERC20. I wasn't aware of the fact that we remove the issuance asset from the FM all together across all FM_BC, and not only for Paribu.